ui/components/ui/currency-display/currency-display.component.js

```diff
@@ -3,6 +3,13 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import { useCurrencyDisplay } from '../../../hooks/useCurrencyDisplay';
 import { EtherDenomination } from '../../../../shared/constants/common';
+import Box from '../box/box';
```

Comment:
Can we try out the new TS version of the `Box`? Component API should be exactly the same so just need to update the import statement 
```suggestion
```